storages/backends/dropbox.py

```diff
@@ -89,6 +93,9 @@ def delete(self, name):
         self.client.files_delete(self._full_path(name))
 
     def exists(self, name):
+        if self.overwrite:
```

Comment:
This is wrong.

Reply:
so how then to make Django to not rename? any suggestions?

Reply:
Look at the other backends. They override `get_available_name`.

Reply:
updated